pywincffi/kernel32/handle.py

```diff
@@ -120,3 +120,50 @@ def WaitForSingleObject(hHandle, dwMilliseconds):
     error_check("WaitForSingleObject")
 
     return result
+
+
+def GetHandleInformation(hObject):
+    """
+    Returns properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724329(v=vs.85).aspx
```

Comment:
For consistency, could you match this url to the others:

```
... seealso::
    
    https://msdn.microsoft.com/en-us/library/ms724329

```

The above url is shorter which means it easier to read.  It also allows us to go back and search/replace all of the msdn references in the future since they'll be identical.